ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -251,6 +251,11 @@ import '../widgets/nuxeo-user-suggestion.js';
           type: String,
           computed: '_computeTitle(shareWithExternal, updatingACE)',
         },
+        _computedUserVisiblePermissions: {
```

Comment:
With this approach `userVisiblePermissions` property seems to be obsolete...
Should this transformation be done on `nuxeo-document-permissions`, since it is there we are passing the property to `nuxeo-popup-permission` (https://github.com/nuxeo/nuxeo-elements/blob/master/ui/nuxeo-document-permissions/nuxeo-document-permissions.js#L103)?

We seem to be using it also on `nuxeo-document-acl-table`, which may make a decision not so easy.

Reply:
Yes, I'll let it like this for now. I don't want to change so much the logic of this property to limit the side effect of this kind of decisions for custom code.

Reply:
Sounds sensible, specifically for the backport. 👍